modules/ecs/main.tf

```diff
@@ -1,5 +1,31 @@
 locals {
   policies = concat([aws_iam_policy.superblocks_agent_policy.arn], var.additional_ecs_execution_task_policy_arns)
+  ecs_container_definition = {
+    name      = "superblocks-agent"
```

Comment:
this should probs be ${var.name_prefix}-agent`

Reply:
I don't want to change this now since this actually would affect people using this module currently.